src/data/directory/lgu/region-i-ilocos-region.json

```
@@ -1105,7 +1094,7 @@
             "name": "ROSITA G. RAFAEL"
           },
           "vice_mayor": {
-            "name": "RODRIGO L. RAFAEL"
+            "name": "APRIL M. SUPNET"
```

```suggestion
            "name": "MARIA LUISA 'APRIL' M. SUPNET"
```

ref: https://lgu201.dilg.gov.ph/view.php?r=01&p=055&m=29